ibft/ibft.go

```diff
@@ -94,7 +99,10 @@ func (i *ibftImpl) Init() {
 	i.processSyncQueueMessages()
 	i.listenToSyncMessages()
 	i.waitForMinPeerOnInit(2) // minimum of 3 validators (me + 2)
-	i.SyncIBFT()
+	if err := i.SyncIBFT(); err != nil {
+		i.logger.Error("crashing.. ", zap.Error(err))
```

Comment:
maybe the error should not be crashing if it doesn't actually crash?

Reply:
OK, will be fixed in the following PR